src/full/Agda/TypeChecking/Quote.hs

```diff
@@ -43,7 +43,7 @@ quotedName = \case
   unambiguous xs
     | Just x <- getUnambiguous xs = return x
     | otherwise =
-        genericError $ "quote: Ambigous name: " ++ prettyShow (unAmbQ xs)
+        genericError $ "quote: Ambiguous name: " ++ prettyShow (unAmbQ xs)
```

Comment:
Thanks!

While you are at it, could you add a testcase that triggers this error?
We do not seem to have one, otherwise the testsuite would fail here now.

Reply:
Done!